src/components/HathorAlert.js

```diff
@@ -5,67 +5,58 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-import React from 'react';
-
+import React, {
+  useState,
+  useEffect,
+  useCallback,
+  forwardRef,
+  useImperativeHandle,
+} from 'react';
 
 /**
  * Component to show an alert on the bottom right corner of the screen
  *
  * @memberof Components
  */
-class HathorAlert extends React.Component {
-  constructor(props) {
-    super(props);
+const HathorAlert = forwardRef(({ text, onDismiss, type, extraClasses }, ref) => {
+  const [show, setShow] = useState(false);
+  const [timer, setTimer] = useState(null);
```

Comment:
No reason, but I don't see any downsides, it will only trigger a re-render when the component dies, but the component death will cause a re-render anyways